backend/ohq/vector_db.py

```diff
@@ -0,0 +1,141 @@
+from django.conf import settings
+import json
+import pinecone
+import openai
+import time
+import tiktoken
+import unidecode
+
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+from ohq.models import Document
+
+openai.api_key = settings.OPENAI_API_KEY
+pinecone.init(api_key=settings.PINECONE_API_KEY, environment='gcp-starter')
```

Comment:
I think u should also include 'gcp-starter' within the global settings object as well